pkg/apis/operator/v1alpha1/common.go

```diff
@@ -173,3 +189,9 @@ type ResourceRequirementsOverride struct {
 	// The desired ResourceRequirements
 	corev1.ResourceRequirements
 }
+
+// Manifest enables the user to specify the links to the manifests' URLs
+type Manifest struct {
+	// The link of the manifest URL
+	Url string `json:"URL"`
```

Comment:
Is this an [`apis.URL`](https://github.com/knative/pkg/blob/master/apis/url.go#L30)?

I think it's not, because it has some form of string replacement supported, right? Should we document the replacement language?

Reply:
That's correct. We should document the string replacement usage.